src/inverted_index.c

```diff
@@ -1299,9 +1253,9 @@ IndexIterator *NewReadIterator(IndexReader *ir) {
 /* Repair an index block by removing garbage - records pointing at deleted documents,
  * and write valid entries in their place.
  * Returns the number of docs collected, and puts the number of bytes collected in the given
- * pointer. If an error occurred - returns -1
+ * pointer.
  */
-int IndexBlock_Repair(IndexBlock *blk, DocTable *dt, IndexFlags flags, IndexRepairParams *params) {
+size_t IndexBlock_Repair(IndexBlock *blk, DocTable *dt, IndexFlags flags, IndexRepairParams *params) {
```

Comment:
Covered somewhere?

Reply:
Yes, by any GC test (we just don't see it today)